cmd/assembly.go

```diff
@@ -35,9 +35,9 @@ var assembleCommand = &cobra.Command{
 
 func init() {
 	assembleCommand.Flags().StringVarP(&assembleCmdSrc,
-		"source", "s", "", "The path to the MTA project; the current path is set as the default")
+		"source", "s", "", "The path to the MTA project; the current path is set as default")
 	assembleCommand.Flags().StringVarP(&assembleCmdTrg,
-		"target", "t", "", "The path to the results folder; the current path is set as the default")
+		"target", "t", "", `The path to the generated MTAR file; the path to the "mta_archives" subfolder of the current folder is set as default`)
```

Comment:
Isn't it the path to the folder in which the MTAR file is created?

Reply:
changed